data/ecosystems/s/stafi.toml

```diff
@@ -0,0 +1,34 @@
+# Ecosystem Level Information
+title = "Stafi"
+
+sub_ecosystems = []
+
+github_organizations = ["https://github.com/stafiprotocol"]
```

Comment:
This is just a github user as opposed to an org, so you can make this an empty array for now.